scripts/JsonRpc.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+
+class JsonRpc(requests.Session):
+    rpc_path = '/json-rpc'
+
+    def __init__(self, address):
+        super(JsonRpc, self).__init__()
+        self.host = address + self.rpc_path
+        self._id = 0
+
+    def request(self, http_method, url, json=None, **kwargs):
```

Comment:
I can't find the documentation for overriding this method. Can you point me to it? I like this design though!